src/EmbedIO/Security/BannedInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Net;
+
+namespace EmbedIO.Security
+{
+    /// <summary>
+    /// Represents the info af a banned IP address.
```

Comment:
"Represents information about the ban of an IP address."

Reply:
I already changed the description.